app/domain/authentication/authn_jwt/signing_key/fetch_provider_uri_signing_key.rb

```diff
@@ -15,17 +16,22 @@ def initialize(
           @discover_identity_provider = discover_identity_provider
 
           @authenticator_input = authenticator_input
+          @fetch_signing_key = fetch_signing_key
+        end
+
+        def call(force_read:)
```

Comment:
Maybe we can align in all palces to refresh instead of force_read?

Reply:
"refresh" or "force_fetch" or "force_read"?

@benoaviram @tzheleznyak

Reply:
the decision is force_fetch